daemon/algod/api/server/router.go

```diff
@@ -97,8 +98,42 @@ func registerHandlers(router *echo.Echo, prefix string, routes lib.Routes, ctx l
 	}
 }
 
+func makeAuthRoutes(apiToken string, adminAPIToken string, enableProfiler bool) middlewares.AuthRoutes {
+	authRoutes := make(middlewares.AuthRoutes)
+
```

Comment:
It would be good to call `tokens.ValidateAPIToken` on `apiToken` and `adminAPIToken` here, just to make absolutely sure we don't serve these endpoints without auth.

Reply:
(I know we call it earlier elsewhere, but it's in a different package, and doing it here might prevent a mistake in the future)